src/services/parameterizer.js

```diff
@@ -8,7 +8,7 @@ class ParameterizerService {
   }
 
   async init () {
-    this.parameterizer = await getParameterizer()
+    this.parameterizer = await getParameterizer(window.web3.eth.defaultAccount)
```

Comment:
@kangarang `window.web3` is undefined if MetaMask isn't installed and will throw error here. Can't assume window.web3 is available

Reply:
@miguelmota fasho. Lemme think about this this afternoon.